ReadMe.md

```diff
@@ -101,7 +101,10 @@ command line parameters on the first run:
 - `install` - build and install all public artifacts into local maven repository
 - `coreLibsTest` - build and run stdlib, reflect and kotlin-test tests
 - `gradlePluginTest` - build and run gradle plugin tests
-- `compilerTest` - build and run all compiler tests
+- `compilerTest` - build and run all compiler tests. These tests are not actively maintained, so only check the tests you are interested in.
```

Comment:
I'm not sure about the wording here. "Not actively maintained" might mean that it's something unimportant or obsolete, which is not the case. Also, how do you imagine we _actively maintain_ the existing hundreds of thousands of tests? :) They exist already, they check the needed behavior, and that's really all that is needed from them.